web-server/v0.4/src/pages/Explore/index.js

```diff
@@ -60,8 +64,40 @@ class Explore extends Component {
     dispatch(routerRedux.push(parsedConfig.routing.location.pathname));
   };
 
+  editDescription = (id, value) => {
+    const { dispatch, datastoreConfig } = this.props;
+    dispatch({
+      type: 'explore/editDescription',
+      payload: { datastoreConfig, id, value },
+    }).then(() => {
+      this.fetchSharedSessions();
+    });
+  };
+
+  showModal = id => {
+    this.setState({
+      visible: true,
+      selectedID: id,
+    });
+  };
+
+  handleSave = () => {
+    const { selectedID } = this.state;
+    const { value } = document.getElementById('editedInput');
```

Comment:
Can we consider using the React API to modify user input using the component state here?